CMakeLists.txt

```diff
@@ -98,6 +98,7 @@ elseif(BASEARCH_PPC_FOUND)
 elseif(BASEARCH_S360_FOUND)
     option(WITH_DFLTCC_DEFLATE "Build with DFLTCC intrinsics for compression on IBM Z" OFF)
     option(WITH_DFLTCC_INFLATE "Build with DFLTCC intrinsics for decompression on IBM Z" OFF)
+    option(WITH_CRC32_VX "Build with vectorized CRC32 on IBM Z" OFF)
```

Comment:
The point of testing with and without specific optimization is to catch errors when the code paths differ... I don't see why we should remove CI instances because of this specific reason.

Reply:
Ok, let turn it `ON` by default and keep the CI but turn it off in CI instance. So long as it is ON by default then anybody who compiles with `WITH_OPTIM` will benefit from the optimization.

Reply:
Personally I have no opinion whether it should be ON or OFF by default... Either is good for me.

Reply:
I'll change the default to ON and the new CI jobs will have it OFF.

Reply:
For such CI jobs, I've usually named them "No AVX2" or "No SSE4" etc..